recOrder/plugin/workers/calibration_workers.py

```diff
@@ -9,16 +9,19 @@
 from recOrder.io.utils import MockEmitter
 from recOrder.calib.Calibration import LC_DEVICE_NAME
 from recOrder.io.metadata_reader import MetadataReader, get_last_metadata_file
+from waveorder.io.writer import WaveorderWriter
 import os
+import shutil
 import numpy as np
 import glob
 import logging
 import json
 
 # type hint/check
 from typing import TYPE_CHECKING
-# avoid circular import error
+# avoid runtime import error
 if TYPE_CHECKING:
+    from _typeshed import StrOrBytesPath
```

Comment:
This is only runs when `TYPE_CHECKING` is `True`, which does not happen at runtime currently. So this is now only a visual hint (just as all the other type hints in vanilla CPython). But if we start to use runtime type checkers such as `pydantic`, this will be used.